### PR TITLE
Fluent announcement

### PR DESCRIPTION
**Pull request by micahgodbolt**
Adds a site side announcement of the recent fabric => fluid blog post. Moves HIG link to resources. 

https://developer.microsoft.com/en-us/office/blogs/ui-fabric-is-evolving-into-fluent-ui/

4c3e0e42-364f-456d-a60c-a5d776543f5e

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12283)

### CODE REVIEW

File: packages/example-app-base/src/components/SiteMessageBar/SiteMessageBar.tsx [hunk 1]
```diff
@@ -28,12 +28,14 @@ export class SiteMessageBar extends React.Component<ISiteMessageBarProps, ISiteM
   }
 
   public render() {
-    const { text, linkUrl, linkText, styles } = this.props;
+    const { text, linkUrl, linkText, styles, ...rest } = this.props;
     const { isVisible } = this.state;
 
+    console.log(text);
```
Comment (Jahnp) [inline packages/example-app-base/src/components/SiteMessageBar/SiteMessageBar.tsx]: Necessary?

File: apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx [hunk 2]
```diff
@@ -58,18 +59,15 @@ export const SiteDefinition: ISiteDefinition<Platforms> = {
   ],
   messageBars: [
     {
-      path: '#/controls/web',
-      text: 'Fabric 7 is now released with the new Fluent styles.',
+      path: '#',
+      text: (
+        <span>
+          <strong style={{ fontWeight: 700 }}>Coming soon</strong> UI Fabric is evolivng into Fluent UI.
```
Comment (ecraig12345) [inline apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx]: Doesn’t strong do this automatically?

File: apps/fabric-website/src/pages/Overviews/ResourcesPage/ResourcesPage.tsx [hunk 3]
```diff
@@ -45,6 +45,15 @@ function _otherSections(): IPageSectionProps[] {
           }
         </Markdown>
       )
+    },
+    {
+      sectionName: 'Microsoft Employees',
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Overviews/ResourcesPage/ResourcesPage.tsx]: Lowercase employees

File: packages/example-app-base/src/components/SiteMessageBar/SiteMessageBar.tsx (hunk 1 above)
Comment (micahgodbolt) [reply packages/example-app-base/src/components/SiteMessageBar/SiteMessageBar.tsx]: hahahahahahaha



....


yes

File: apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx (hunk 2 above)
Comment (micahgodbolt) [reply apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx]: it only did weight 600, which wasn't very different

**Pull request merged**